migrations/20190107123192-create-comment.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('comments', {
```

Comment:
We agreed on the first character of the Model being uppercase. Please change this.

Reply:
Ok. Right away